### PR TITLE
Minor spelling/grammatical changes

### PR DESCRIPTION
**Pull request by CSol314**

### CODE REVIEW

File: framework/doc/resources.rst [hunk 1]
```diff
@@ -66,10 +50,10 @@ This example could be enhanced to dynamically react to bundles being
 started and stopped, making use of the popular *extender pattern* from
 OSGi.
 
-Runtime overhead
+Runtime Overhead
 ----------------
 
-The resources system has the following runtime characteristics:
+The resource has the following runtime characteristics:
```
Comment (jeffdiclemente) [inline framework/doc/resources.rst]: "resource" doesn't seem to be the right word considering what follows describes the entire resource system as a whole.

File: framework/doc/resources.rst [hunk 2]
```diff
@@ -79,3 +63,18 @@ The resources system has the following runtime characteristics:
 -  Creating a ``BundleResourceStream`` object will allocate memory for
    the uncompressed resource data and inflate it. The memory will be
    free'ed after the ``BundleResourceStream`` object is destroyed.
+   
+ Conventions and Limitations
+ ---------------------------
+ 
+-  Resources have a size limitation of 2GB, due to the used ZIP format.
```
Comment (jeffdiclemente) [inline framework/doc/resources.rst]: "due to the used ZIP format." sounds awkward to me.

File: framework/doc/resources.rst [hunk 3]
```diff
@@ -79,3 +63,18 @@ The resources system has the following runtime characteristics:
...
+-  Resource entries are stored with case-insensitive names. On
+   case-sensitive file systems, adding resources with the same name but
+   different capitalization will lead to an error.
+-  Looking up resources by name at runtime *is* case sensitive.
+-  The CppMicroServices library will search for a valid zip file inside
+   a shared library, starting from the end of the file. If other zip
+   files are embedded in the bundle as well (e.g. as an additional
+   resource embedded via the Windows RC compiler or using other
+   techniques), it will stop at the first valid zip file and use it a
```
Comment (jeffdiclemente) [inline framework/doc/resources.rst]: did you mean "use it as" instead of "use it a"?

File: framework/doc/static_bundles.rst [hunk 4]
```diff
@@ -5,13 +5,16 @@ Static Bundles
 
 The normal and most flexible way to add a CppMicroServices bundle to
 an application is to compile it into a shared library that is later
-:any:`installed <cppmicroservices::BundleContext::InstallBundles>` in the application. 
+installed programmatically using the command
```
Comment (jeffdiclemente) [inline framework/doc/static_bundles.rst]: command isn't the best word to describe the function. "API" is a better one.

File: framework/doc/static_bundles.rst [hunk 5]
```diff
@@ -5,13 +5,16 @@ Static Bundles
 
 The normal and most flexible way to add a CppMicroServices bundle to
 an application is to compile it into a shared library that is later
-:any:`installed <cppmicroservices::BundleContext::InstallBundles>` in the application. 
+installed programmatically using the API 
+:any:`installed <cppmicroservices::BundleContext::InstallBundles>` at
+runtime.
```
Comment (saschazelzer) [inline framework/doc/static_bundles.rst]: I suggest to write "using the :any:`BundleContext::InstallBundles() <cppmicroservices::BundleContext::InstallBundles>` function at runtime" instead.

File: framework/doc/service_hooks.rst [hunk 6]
```diff
@@ -14,9 +14,9 @@ application bundles.
 
 Some example use cases for service hooks include:
 
-- Proxying of existing services by hiding the original service and 
-registering a *proxy service* with the same properties 
-- Providing services *on demand* based on registered service listeners from external bundles
+   - Proxying of existing services by hiding the original service and 
+     registering a *proxy service* with the same properties 
+   - Providing services *on demand* based on registered service listeners from external bundles
```
Comment (saschazelzer) [inline framework/doc/service_hooks.rst]: Now there is too much indentation and the block [renders as a quote](https://github.com/CSol314/CppMicroServices/blob/2cd6b067dc5a56597607fd8ae7de6bb9145a2940/framework/doc/service_hooks.rst). Do not indent the lines starting with a `-`.

File: framework/doc/static_bundles.rst [hunk 7]
```diff
@@ -4,10 +4,7 @@ Static Bundles
 ==============
 
 The normal and most flexible way to add a CppMicroServices bundle to
-an application is to compile it into a shared library that is later
-installed programmatically using the API 
-:any:`installed <cppmicroservices::BundleContext::InstallBundles>` at
-runtime.
+an application is to compile it into a shared library using the :any:BundleContext::InstallBundles() <cppmicroservices::BundleContext::InstallBundles> function at runtime.
```
Comment (saschazelzer) [inline framework/doc/static_bundles.rst]: Now the backticks `` ` `` are missing. It needs to be `` :any:`text <link>` ``. I guess they got lost in my earlier comment due to the GitHub formatting.

File: framework/doc/service_hooks.rst (hunk 6 above)
Comment (CSol314) [inline framework/doc/service_hooks.rst]: When you mentioned "missing indentation", could you possibly clarify what you mean by that? I thought you meant add an indentation, but I think I am misunderstanding you.

**Comment by saschazelzer** [comment]
Thanks! I added two small corrections before merging.

**Pull request merged**